pages/tos/index.vue

```diff
@@ -0,0 +1,17 @@
+<template>
+  <div class="terms-of-service">
+    Terms of Service
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'TermsOfService'
```

Comment:
For SEO, a page's route should be as descriptive as possible. For this reason, i would suggest changing this route from `tos` to `terms-of-service`.